recipes/python-pdal/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "python-pdal" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "60d41da79441d0aa9c44ec6ebb7497ffc0090d761259d42d4eeb05c7146bb432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/PDAL/python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and py<35]
```

Comment:
@dougalsutherland I'd like to get this out sooner rather than later. What are your feelings on reverting the skip statement back to `py!=36` while we wait for https://github.com/conda-forge/pdal-feedstock/pull/24 and upstream issues to resolve themselves.

Reply:
Let's try to get the upstream thing resolved first: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/55.

Reply:
Any more clarity on whether or not the pinning PR is going to move forward? Trying to gauge the likelihood that I'll be able to present this new package at a talk I'm giving next week (hence the eagerness :wink:).

Reply:
Sorry for the slowness, I think the relevant people happen to be pretty swamped right now. I just released the pinning PR, and am updating https://github.com/conda-forge/pdal-feedstock/pull/24 now.